scripts/README.rst

```diff
@@ -0,0 +1,84 @@
+This module provides scripts to update the STM32Cube for Zephyr.
+Follow below steps to update the STM32Cube HAL from the latest
+version found in STM32CubeXX repositories.
+################################################################
```

Comment:
Title is quite long for a document header, should likely be part of introduction

Reply:
Reformatted